### PR TITLE
SAK-47559: Improve graphical display of test statistics (removed by error)

### PR DESCRIPTION
**Pull request by jesusmmp**
https://sakaiproject.atlassian.net/browse/SAK-47559

This piece of code was removed by error on: https://sakaiproject.atlassian.net/browse/SAK-47139
it was introduced here: https://sakaiproject.atlassian.net/browse/SAK-42179

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 1]
```diff
@@ -1630,12 +1630,16 @@ private void getTFMCScores(Map publishedAnswerHash, List scores, HistogramQuesti
...
 			int correctresponses = 0;
+			int barsFilteredSize = 0;
 	
 			// find answers sorted by sequence
 			while (iter.hasNext()) {
 				Long sequenceId = (Long) iter.next();
 				Long answerId = (Long) sequenceMap.get(sequenceId);
 				AnswerIfc answer = (AnswerIfc) texts.get(answerId);
+
+				if (answer.getText() == null || "".equals(answer.getText())) continue;
```
Comment (ottenhoff) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:1641]: if (StringUtils.isBlank(answer.getText())

Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:1641]:
```suggestion
				if (StringUtils.isBlank(answer.getText()) continue;
```

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java [hunk 2]
```diff
@@ -1630,12 +1630,16 @@ private void getTFMCScores(Map publishedAnswerHash, List scores, HistogramQuesti
...
 			int correctresponses = 0;
+			int barsFilteredSize = 0;
 	
 			// find answers sorted by sequence
 			while (iter.hasNext()) {
 				Long sequenceId = (Long) iter.next();
 				Long answerId = (Long) sequenceMap.get(sequenceId);
 				AnswerIfc answer = (AnswerIfc) texts.get(answerId);
+
+				if (StringUtils.isBlank(answer.getText()) continue;
```
Comment (ern) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java:1641]:
```suggestion
				if (StringUtils.isBlank(answer.getText())) continue;
```

**Comment by jesusmmp** [comment]
Thanks @ern !

**Pull request merged**